src/components/StepTracker.jsx

```diff
@@ -0,0 +1,19 @@
+import Step from "./Step";
+
+const StepTracker = ({ steps, activeStep }) => {
+  return (
+    <>
+      {steps.map((step) => {
+        return (
+          <Step
```

Comment:
add the key prop to the <Step>

Reply:
There's a whole thing why you shouldn't use index as the key. Just use the labels for now, but typically you would use something unique.